.env.sample

```diff
@@ -149,15 +149,16 @@ DEV_AUTH=false
 
 # Optional overrides to force hitting remote server instead of local
 # These 3 should usually be set for frontend development
-LOCAL_API_URL="https://api.sandbox.jicloud.org"
-LOCAL_UPLOADS_URL="https://uploads.sandbox.jicloud.org"
-LOCAL_MEDIA_URL="https://media.jicloud.org"
-LOCAL_LEGACY_URL="https://legacy.jicloud.org"
+LOCAL_API_URL="https://api.sandbox.jigzi.org"
+LOCAL_UPLOADS_URL="https://uploads.sandbox.jigzi.org"
+LOCAL_MEDIA_URL="https://media.jigzi.org"
+LOCAL_LEGACY_URL="https://legacy.jigzi.org"
```

Comment:
`LOCAL_API_URL` change is great!

Others `LOCAL_UPLOADS_URL`, `LOCAL_MEDIA_URL`, and `LOCAL_LEGACY_URL` are using `jicloud` (we should probably change that but for now that's the case)

Reply:
Cool, that was a poor assumption on my part - I hadn't tested that those values worked yet for me.